### ISSUE TITLE
Website needs like to Github

### ISSUE THREAD

**Issue by alerque** [UberWriter/uberwriter#162]
The website needs to link to the Github project in some way.

**Comment by somas95** [comment]
Yep, that's true
For the next release I want to ship out a new website with more information and a better design overall

**Comment by alerque** [comment]
I understand the `gh-pages` thing, but given that you have a Github group for the project already it might be worth splitting the website out into a different repo. Issues would be easier to track that way too ;-)

**Comment by somas95** [comment]
Good point, and probably it would be a good idea to split out the localizations, but I still have to do it properly (we have them integrated with Poeditor)
